src/components/molecule_word_card/index.shared.tsx

```diff
@@ -41,7 +43,13 @@ const WordCardShared: FC<Props> = ({ wordId }) => {
 
   return (
     <StyledSuspense>
-      <Card style={{ width: `100%`, borderRadius: 9 }}>
+      <Card
+        style={{
+          width: width > 1000 ? `1000px` : `100%`,
+          maxWidth: `1000px`,
+          borderRadius: 9,
+        }}
+      >
```

Comment:
dont forget to delete

Reply:
Handled in [65825d0](https://github.com/ajktown/wordnote/pull/205/commits/65825d09cd61c7debbdc17e9cffc23c128200fb6)